utoipa-swagger-ui/src/rocket.rs

```diff
@@ -2,12 +2,11 @@
 
 use std::{borrow::Cow, io::Cursor, sync::Arc};
 
+use rocket::response::status::Custom;
+use rocket::response::Responder;
```

Comment:
this is imported twice, here as `Responder` and below as `RocketResponder`.  That is confusing.  IMO keep the existing `RocketResponder` and remove this import, and also remove `use rocket::response::status::Custom;` as these are unnecessary changes which change the blame of multiple lines that did not actually change.

Reply:
I applied the suggested changes!